src/components/ebay-infotip/template.marko

```diff
@@ -6,9 +6,11 @@
         w-on-base-expand="handleExpand"
         w-on-base-collapse="handleCollapse">
         <span
+            w-preserve-attrs="class,id"
```

Comment:
Presently, preserving the class doesn't do much since the class is always just `"infotip"`. Any HTML class attribute passed in is never used (lines 10-11 👇). @melquan and I uncovered this and I've created an issue for it: #604.

Reply:
Makeup expander is what is changing the class which is why I've added this.

Reply:
Ah, right. Still we have the same problem on the HTML attribute, though.